src/main/java/com/suse/saltstack/netapi/parser/SaltStackParser.java

```diff
@@ -25,6 +26,8 @@
         new SaltStackParser<>(new TypeToken<SaltStackResult<List<SaltStackToken>>>(){});
     public static final SaltStackParser<SaltStackResult<List<SaltStackJob>>> JOB =
         new SaltStackParser<>(new TypeToken<SaltStackResult<List<SaltStackJob>>>(){});
+    public static final SaltStackParser<SaltStackResult<List<HashMap>>> RETVALS =
+        new SaltStackParser<>(new TypeToken<SaltStackResult<List<HashMap>>>(){});
```

Comment:
Yes, I agree, let's make it `Map<String, Object>` here and above. After that it can be merged (when the indent has been fixed as well ;)).